gulpfile.js

```diff
@@ -1,14 +1,34 @@
 var copy = require("copy"),
     clear = require("cli-clear"),
     del = require("del"),
+    exec = require("child_process").exec,
     gulp = require("gulp"),
-    gutil = require("gulp-util");
+    gutil = require("gulp-util"),
+    runSequence = require("run-sequence");
 
 gulp.task("clear", function(cb) {
     clear();
     cb();
 });
 
+gulp.task("clean", function(cb) {
+    del("./coverage")
+    .then(function(files) {
+        if(files.length > 0) {
+            gutil.log("Deleted %s", files);
```

Comment:
I wouldn't worry about logging this, personally

Reply:
The reason I'm cleaning up the coverage folder is that I want to know that I'm always looking at newly generated coverage when I'm developing, and this log statement gives me a visual clue that the coverage folder actually has been wiped, so I'd rather keep this ☺️

Reply:
I do agree that it's useful to have confirmation the files are being deleted while working on the tooling - gotta make sure it's working properly. However, once we're confident in the build (and what the build does), perhaps the extra logging will just become noise?

Reply:
Removed!